types/google.visualization/index.d.ts

```diff
@@ -654,6 +703,11 @@ declare namespace google {
             reverseCategories?: boolean;
             selectionMode?: string // single / multiple
             series?: any;
+            domainAxis?: { type: string };
+            trendlines?: { [key: number]: LineChartTrendlineOptions; };
```

Comment:
Try if it works:
`trendlines?: Record<number, LineChartTrendlineOptions>;

Reply:
Yea, that works, one moment on the commit.

Reply:
Reverted